### PR TITLE
fix world loading

### PR DESCRIPTION
**Pull request by DasCapschen**

### CODE REVIEW

File: src/project.godot [hunk 1]
```diff
@@ -208,10 +208,6 @@ ContentLoader="*res://Data/Scripts/Singletons/content_loader.gd"
 LoadingScreen="*res://Data/UI/loading_screen.tscn"
 InputHelper="*res://Data/Scripts/Singletons/input_helper.gd"
 
-[debug]
-
-settings/stdout/verbose_stdout=true
-
```
Comment (HaSa1002) [inline src/project.godot]: 🤨

**Review by HaSa1002 — APPROVED** [review]

File: src/project.godot (hunk 1 above)
Comment (DasCapschen) [reply src/project.godot]: fixed.

Comment (HaSa1002) [reply src/project.godot]: I know you hate that option. We should be able to get rid of that soon-ish

**Comment by HaSa1002** [comment]
thank you

**Pull request merged**